File: River/River.Droid/Assets/Packages/Ahoy.Net/Scripts/Messages/MessageEventArgs.cs

using UnityEngine;
using System;
using System.Net;
using System.Net.Sockets;
using System.Collections.Generic;

namespace Ahoy.Net
{


    public struct MessageEventArgs
    {
        public EndPoint sender;
        public AhoySocket receiver;
        public Message message;

        public MessageEventArgs(EndPoint _sender, AhoySocket _receiver, Message _message)
        {
            receiver = _receiver;
            sender = _sender;
            message = _message;
        }



        public override string ToString()
        {
            return $"sender:{sender}, receiver: {receiver}, message:\n{message}";
        }


    }
}

File: River/River.Body/Ahoy/Messages/MessageEventArgs.py


class MessageEventArgs():
    def __init__(self, sender, receiver, message):
        self.sender = sender
        self.receiver = receiver
        self.message = message

    def __str__(self):
        return "sender: {}, receiver: {}, message: \n{}".format(self.sender, self.receiver,  self.message)
# $"receiver: {receiver}, sender:{sender}, data length:{data.Count}, data:{data.ElementsToString()}";
